test/e2e/node/kubelet_perf.go

```diff
@@ -110,9 +110,9 @@ func runResourceTrackingTest(f *framework.Framework, podsPerNode int, nodeNames
 	framework.PrintPerfData(framework.ResourceUsageToPerfData(rm.GetMasterNodeLatest(usageSummary)))
 	verifyMemoryLimits(f.ClientSet, expectedMemory, usageSummary)
 
-	cpuSummary := rm.GetCPUSummary()
-	framework.Logf("%s", rm.FormatCPUSummary(cpuSummary))
+	rm.LogCPUSummary()
```

Comment:
Humm, this change makes GetCPUSummary() call twice.
So I feel now it is better to keep this itself without changes.